doc/095_considerations.rst

```diff
@@ -0,0 +1,35 @@
+..
+  Normally, there are no heading levels assigned to certain characters as the structure is
+  determined from the succession of headings. However, this convention is used in Python’s
+  Style Guide for documenting which you may follow:
+
+  # with overline, for parts
+  * for chapters
+  = for sections
+  - for subsections
+  ^ for subsubsections
+  " for paragraphs
+
+######################
+Special Considerations
+######################
+
+************************************
+Large Repositories and Min Pack Size
+************************************
+
+In certain instances, such as very large repositories, it is desired to have larger pack
+sizes to reduce the number of files in the repository.  Notable examples are OpenStack
+Swift and some Google Drive Team accounts.  This can be achieved by either using the
+``--min-packsize`` flag or defining the ``$MIN_PACKSIZE`` environment variable.
```

Comment:
RESTIC_MIN_PACKSIZE, not MIN_PACKSIZE. Also, Swift and Drive Team are not example of very large repositories. I suppose they're examples of systems where having many files is inefficient?

Reply:
Oh, I should have read the forum post first. They're examples of hard limits on the number of files. I think that should be explained.

Reply:
Thanks for catching that!

I've fixed the env variable, and clarified reasons min pack size would want to be higher.